src/pretix/control/views/waitinglist.py

```diff
@@ -47,16 +47,19 @@
 from django.utils.timezone import now
 from django.utils.translation import gettext_lazy as _, pgettext
 from django.views import View
-from django.views.generic import ListView
-from django.views.generic.edit import DeleteView
-
+from django.views.generic import ListView, FormView
+from django.views.generic.edit import DeleteView, UpdateView
```

Comment:
Do not import UpdateView from django, import it from . like https://github.com/pretix/pretix/blob/master/src/pretix/control/views/discounts.py#L48 – that way you get `get_form_kwargs` through inheritance as described further down.